src/routes/authRoutes.ts

```diff
@@ -0,0 +1,17 @@
+import { Hono } from 'hono';
+import { authController } from '@/controllers/authController.ts';
+import { errorHandler } from '@/utils/custom-error';
+import { validateSign } from '@/middlewares/authValidation';
+import { jwt } from 'hono/jwt';
+
+export const authRoutes = new Hono().basePath('/auth');
+
+authRoutes.onError(errorHandler);
```

Comment:
If you use the same error handler for all the routes, then please put it in the `routes/index.ts`